dashboard/app/controllers/plc/enrollment_evaluations_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Plc::EnrollmentEvaluationsController < ApplicationController
+  def perform_evaluation
+    authorize! :read, Plc::Course
+    @plc_user_course_enrollment = Plc::UserCourseEnrollment.find(params[:enrollment_id])
+    @questions = @plc_user_course_enrollment.plc_course.plc_evaluation_questions.all
```

Comment:
in general it is nice to not dereference (using `.all`) the activerecord collection proxies until you actually need them. Right now it's the same because it will happen when you call each on it in the view -- but if your view ends up changing to not always use (a)questions, you can save a query.